pkg/opensearch/ppl_response_parser_test.go

```diff
@@ -295,18 +297,18 @@ func TestPPLResponseParser(t *testing.T) {
 				response := `{
 							"schema": [
 								{ "name": "timeName", "type": "timestamp" },
-								{ "name": "testMetric", "type": "string" }
+								{ "name": "testMetric", "type": "integer" }
```

Comment:
With an invalid value type, this and the test below weren't actually getting to the part of the code that tests the timestamp